spk/aria2/src/installer.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+
+# Package
+PACKAGE="aria2"
+DNAME="aria2"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PATH="${INSTALL_DIR}/bin:/usr/local/bin:/bin:/usr/bin:/usr/syno/bin"
+USER="aria2"
+GROUP="users"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+WEB_DIR="/var/services/web"
+LOG_FILE="${INSTALL_DIR}/var/aria2.log"
+CFG_FILE="${INSTALL_DIR}/var/aria2.conf"
+
+# SERVICETOOL="/usr/syno/bin/servicetool"
+# FWPORTS="/var/packages/${PACKAGE}/scripts/${PACKAGE}.sc"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Install the web interface
+    cp -pR ${INSTALL_DIR}/share/${PACKAGE} ${WEB_DIR}
+
+     # Install busybox stuff
+    ${INSTALL_DIR}/bin/busybox --install ${INSTALL_DIR}/bin
+
+    # Create user
+    adduser -h ${INSTALL_DIR}/var -g "${DNAME} User" -G ${GROUP} -s /bin/sh -S -D ${USER}
+
+    # Edit the configuration according to the wizard
+    sed -i -e "s|@download_dir@|${wizard_download_dir:=/volume1/downloads}|g" ${CFG_FILE}
+    sed -i -e "s/@username@/${wizard_control_username:=admin}/g" ${CFG_FILE}
+    sed -i -e "s/@password@/${wizard_control_password:=admin}/g" ${CFG_FILE}
+    sed -i -e "s|@log_file@|${LOG_FILE}|g" ${CFG_FILE}
```

Comment:
Maybe I'm not seeing it, but is there a reason you `sed` the value, instead of just adding it directly to the config file?